schemas/json-schema-validation.spec.js

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Why the blank lines at the top of various files?

Reply:
no reason, I tend to write spacy code... this should be resolved with a linter IMO.